### DISCUSSION TITLE
Unable to publish my site

### DISCUSSION THREAD

**Discussion by Laura05010** [alshedivat/al-folio#2743] [Q&A] [answered]
Although the site perfectly runs on docker locally, I'm unable to deploy it. 
master and gh-pages keep failing especifically with the prettier-formatter: 

Run npx prettier . --check
Checking formatting...
[warn] _includes/art.liquid
[warn] _pages/about.md
[warn] _pages/art.md
[warn] _pages/projects.md
[warn] _pages/teaching.md
[warn] _posts/2021-07-20-technically-speaking.md
[warn] _posts/2022-11-07-social-media-addictions.md
[warn] _projects/1_art.md
[warn] _projects/2_art.md
[warn] _projects/3_art.md
[warn] _sample_posts/201[5](https://github.com/Laura05010/al-folio/actions/runs/11155502712/job/31006533362#step:5:6)-10-20-math.md
[warn] _sample_posts/2018-12-22-distill.md
[warn] _sass/_base.scss
[warn] _sass/_variables.scss
[warn] assets/js/theme.js
[warn] README.md
[warn] Code style issues found in 1[6](https://github.com/Laura05010/al-folio/actions/runs/11155502712/job/31006533362#step:5:7) files. Run Prettier to fix.
Error: Process completed with exit code 1.

so because of this i'm unable to deploy on netlify. 
my repo is public, you can check it out here to see exactly what I mean : https://github.com/Laura05010/al-folio

**Comment by george-gca** [comment, marked as answer]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-code-runs-fine-locally-but-when-i-create-a-commit-and-submit-it-it-fails-with-prettier-code-formatter-workflow-run-failed-for-master-branch-how-do-i-fix-that). Also duplicate of #2623, #2178.
